blazor/treegrid/editing/template-editing.md

```diff
@@ -140,4 +137,272 @@ public class TreeData
 
 ![Blazor TreeGrid with Dialog Template](../images/blazor-treegrid-dialog-template.png)
 
-> The template form editors should have **name** attribute.
\ No newline at end of file
+> The template form editors should have **name** attribute.
+
+### Disable components in dialog template
+
+It is possible to disable particular components rendered inside the dialog template using the data source value. This can be achieved by utilizing the `Enabled` property of the components which specifies whether the component is enabled or disabled.
+
+This is demonstrated in the following sample code, where if the `RequestType` argument value of the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeGrid.TreeGridEvents-1.html#Syncfusion_Blazor_TreeGrid_TreeGridEvents_1_OnActionBegin) event is **BeginEdit** then the `Enabled` property of the **TaskId** and **TaskName** Textbox is set to false.

```

Comment:
This is demonstrated in the following sample code, where if the `RequestType` argument value of the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeGrid.TreeGridEvents-1.html#Syncfusion_Blazor_TreeGrid_TreeGridEvents_1_OnActionBegin) event is **BeginEdit**, the `Enabled` property of the **TaskId** and **TaskName** Textbox is set to false.

Reply:
resolved